### PR TITLE
Adding Justdubs.org

### PR DESCRIPTION
**Pull request by nate-moo**
<sub><sub>plz dont smite me on my first PR</sub></sub>

### CODE REVIEW

File: anime_downloader/sites/justdubs.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        for a in soup:
+            for b in servers:
+                if servers[b] in a.get('src'):
+                    return [(b, a.get('src'))]
+
+        logger.debug('Unsuported url, trying mp4upload')
+        link = [('mp4upload',soup[0].get('src'))] if len(soup) != 0 else ''
+        return link
+
+#class JustDubsEpisode(AnimeEpisode, sitename='justdubs'):
```
Comment (Blatzar) [inline anime_downloader/sites/justdubs.py:58]: You might want to delete these comments

File: anime_downloader/sites/justdubs.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        servers =  {
+        'mp4upload':'https://mp4upload.com/',
+        'gcloud':'https://gcloud.live/'
+        }
+        soup = helpers.soupify(helpers.get(self.url)).select('iframe')
+        #link = soup.find_all('iframe')
+        
+        for a in soup:
+            for b in servers:
+                if servers[b] in a.get('src'):
```
Comment (Blatzar) [inline anime_downloader/sites/justdubs.py:51]: You could probably add another loop here for preferred servers in config

File: anime_downloader/sites/justdubs.py (hunk 1 above)
Comment (vn-ki) [inline anime_downloader/sites/justdubs.py:58]: Remove this

**Comment by nate-moo** [comment]
Removed comments and fixed lines

File: anime_downloader/sites/justdubs.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                    return [(b, a.get('src'))]
+
+        return link
```
Comment (Blatzar) [inline anime_downloader/sites/justdubs.py:54]: Will give undefined error if this is reached.

Comment (nate-moo) [reply anime_downloader/sites/justdubs.py:54]: weird, my local has 
`for a in soup:
            for b in servers:
                if servers[b] in a.get('src'):
                    return [(b, a.get('src'))]

        logger.debug('Unsuported url, trying mp4upload')
        return link`

**Comment by nate-moo** [comment]
^ lagrad's changes in gcloud for enabling fembed ^

**Pull request merged**